code/workspaces/client-api/src/config/database.js

```diff
@@ -33,7 +33,7 @@ function createConnection() {
     pass: config.get('databasePassword'),
     auth: { authSource: 'admin' },
   };
-  mongoose.connect(`mongodb://${config.get('databaseHost')}/datalab`, options);
```

Comment:
Just wondering why you need to specify a collection, is this a requirement? Seems a bit arbitrary.

Reply:
It is a requirement - you have to give a database name, not just a host.  However, it did get me wondering which was the appropriate database for the client-api... and it turns out the client api no longer makes any database calls!  So more stuff gotten rid of :-)